infra/README.md

```diff
@@ -38,17 +38,17 @@ Commands:
   gke cluster delete
     gke cluster delete -a service-account.json -f FileOrFolder
 
-  gke nodepool create
-    gke nodepool create -a service-account.json -f FileOrFolder
+  gke nodes create
```

Comment:
To reduce the size of this PR, can you make the GKE specific changes in a separate PR? It'll also make things more clearer. I'll just mention the places you could move to a GKE specific PR in following comments. :)